ChronosTankDrive/src/main/java/frc/robot/Robot.java

```diff
@@ -127,28 +132,39 @@ public void robotInit() {
     rightBackDrive.restoreFactoryDefaults();
     leftEncoder.setPosition(0);
     rightEncoder.setPosition(0);
-    /*
-     * autoChoices.setDefaultOption("AutoLine", "AutoLine");
-     * autoChoices.addOption("BallPickup", "BallPickup");
-     * autoChoices.addOption("Shoot", "Shoot");
-     */
+    autoChoices.setDefaultOption("Shoot1", "Shoot1");
+    autoChoices.addOption("Shoot2", "Shoot2");
+    SmartDashboard.putData(autoChoices);
     rightFrontDrive.setInverted(true);
     leftBackDrive.follow(leftFrontDrive);
     rightBackDrive.follow(rightFrontDrive);
-    limelight=new Limelight(24,32+3,9.5,16.5);
-    chronosDrive = new DifferentialDrive(leftFrontDrive,rightFrontDrive);
-    intake=new Intake(INTAKE_MOTOR_PORT, INTAKE_DEPLOY_PORT, weaponStick);
-    shooter=new ShooterSupersystem(new Shooter(SHOOT_PORT,KICKER_PORT,.01,leftStick), new Hood(HOOD_PORT,HOOD_LIMIT_PORT,leftStick), limelight,chronosDrive, leftStick);
-    subSystems=new Subsystem[]{intake,shooter,new Climber(8, 10, weaponStick)};
-    for(Subsystem subSystem :subSystems){
+    limelight = new Limelight(24, 32 + 3, 9.5, 16.5);
+    chronosDrive = new DifferentialDrive(leftFrontDrive, rightFrontDrive);
+    intake = new Intake(INTAKE_MOTOR_PORT, INTAKE_DEPLOY_PORT, weaponStick);
+    shooter = new ShooterSupersystem(new Shooter(SHOOT_PORT, KICKER_PORT, .01, leftStick),
+        new Hood(HOOD_PORT, HOOD_LIMIT_PORT, leftStick), limelight, chronosDrive, leftStick, weaponStick);
+    subSystems = new Subsystem[] { intake, shooter, new Climber(8, 10, weaponStick) };
+    for (Subsystem subSystem : subSystems) {
       subSystem.init();
     }
   }
-  public void accelLimit(double leftInput,double rightInput){
-    rightAdjusted=(1/accelDriveKonstant)*rightInput+(accelDriveKonstant-1)/accelDriveKonstant*rightAdjusted;
-    leftAdjusted=(1/accelDriveKonstant)*leftInput+(accelDriveKonstant-1)/accelDriveKonstant*leftAdjusted;
-    //chronosDrive.tankDrive(leftAdjusted, rightAdjusted);
+
+  public void accelLimit(double leftInput, double rightInput) {
+    rightAdjusted = (1 / accelDriveKonstant) * rightInput
+        + (accelDriveKonstant - 1) / accelDriveKonstant * rightAdjusted;
+    leftAdjusted = (1 / accelDriveKonstant) * leftInput + (accelDriveKonstant - 1) / accelDriveKonstant * leftAdjusted;
+    // chronosDrive.tankDrive(leftAdjusted, rightAdjusted);
+  }
+
+  public void teleopInit() {
+    shooter.getHood().home();
```

Comment:
@TonyBaston  I'm worried that this could cause the robot to hang at the start of teleop if the hood for some reason didn't home correctly. It may be better to add the non-blocking version to teleopPeriodic() so that they can still drive while it is homing if needed.